docker-compose.yml

```
@@ -1,8 +1,8 @@
 version: "3.4"
 services:
-  nethermind-xdai.dnp.dappnode.eth:
+  nethermind:
     build:
-      context: ./build
+      context: .
       args:
         UPSTREAM_VERSION: 1.14.5
```

```suggestion
        UPSTREAM_VERSION: 1.14.6
```
Needed upgrade for Gnosis Chain

---

The version must be tested before